lib/Listener/AllowBlobMediaInCSPListener.php

```diff
@@ -27,6 +27,7 @@ public function handle(Event $event): void {
 
 		$csp = new ContentSecurityPolicy();
 		$csp->addAllowedMediaDomain('blob:');
+		$csp->addAllowedWorkerSrcDomain('blob:');
```

Comment:
Why is it needed?

Reply:
compression / decompression is done in webworkers which use the blob schema.

Reply:
Added a code comment